core/src/main/res/values/dev_menu_strings.xml

```diff
@@ -35,5 +35,6 @@
     <string name="copy_issue_text">Copy to Clipboard</string>
     <string name="file_issue_warning_text">
         Warning: device and environment information will be included in the issue description.
```

Comment:
re @richardmcclellan's observation - we could remove the explicit "Warning: " label, too? What's left will certainly still _be_ and _act_ as a warning, it just won't trigger the same psychological alarm, if we remove the declaration.
```suggestion
        Device and environment information will be included in the issue description.
```